README.md

```diff
@@ -1,24 +1,88 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Exclusive-Clubhouse(Members-Only)
 
-Things you may want to cover:
+![screenshot](./app/assets/images/clubhse1.png)
 
-* Ruby version
+This project puts your knowledge of authentication, sessions and cookies as well as forms, models, controllers, and validations to the tests. The aim of the project is to build a Exclusive Clubhouse that allows users to see the identity of authors of posts but outsiders just view stories but authors remain anonymous, using devise and Rails methods/helpers.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby -v 2.7.0
+- Rails -v 6.1.1
+- Live-Reloader
+- Bulma(CSS)
+- SQLite3
+- Devise
+- Guard
 
-* Database creation
+## Getting Started
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+To get this project up and running locally, you must already have ruby and rails plus the necessary gems installed on your computer
 
-* Services (job queues, cache servers, search engines, etc.)
+**simple steps set up on your local machine**
 
-* Deployment instructions
+```
+- $ git clone `$ git clone https://github.com/jebitok-dev/Members_only`
+- $ git checkout develop
+- Run `cd Twitter-clone`
```

Comment:
I think you meant, Run `cd Members_only` kindly update the correct path.

Reply:
I have read and somehow understood the No Logic in views but I'm having a hard time converting the ones available into the helpers file since it's linked to the devise method "signed_in?"

Reply:
See an [example implementation](https://github.com/acushlakoncept/eventbrite-lite/blob/master/app/helpers/application_helper.rb), perhaps you may learn one or two: